PSReadLine/PSReadLine.csproj

```diff
@@ -21,7 +21,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
-    <PackageReference Include="System.Management.Automation" Version="7.1.0-rc.2" />
+    <PackageReference Include="System.Management.Automation" Version="7.1.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.PowerShell.Pager" version="1.0.0-preview.4" />
```

Comment:
Please update the `- template: EsrpSign.yml@ComplianceRepo` section of the [releaseBuild.yml](https://github.com/PowerShell/PSReadLine/blob/master/.vsts-ci/releaseBuild.yml) to exclude this assembly from signing.

Reply:
Added it